test/module/shared_model/validators/proposal_validator_test.cpp

```diff
@@ -6,6 +6,8 @@
 #include "module/shared_model/validators/validators_fixture.hpp"
 
 #include <gtest/gtest.h>
+#include <builders/protobuf/transaction.hpp>
```

Comment:
```suggestion
#include "builders/protobuf/transaction.hpp"
```
Since it is not a library or system include. Same for the line below.

Reply:
Fixed